lib/transform.js

```diff
@@ -31,6 +42,19 @@ function isString(y) {
   return typeof y === 'string';
 }
 
+function generateComments(syntax){
+  var commentStart = commentTypes[syntax]?.start || commentTypes.md.start
+  , commentEnd = commentTypes[syntax]?.end || commentTypes.md.end
```

Comment:
Simplify as `syntax = 'md'` and then call `commentTypes[syntax]`, no need to fallback safely